Formula/p/pypy3.9.rb

```diff
@@ -135,8 +140,16 @@ def post_install
     # Symlink the prefix site-packages into the cellar.
     site_packages(libexec).parent.install_symlink site_packages(HOMEBREW_PREFIX)
 
-    # Tell distutils-based installers where to put scripts
+    # Create a scripts folder in the prefix and symlink it as libexec/bin.
+    # This is needed as setuptools' distutils ignores our distutils.cfg.
+    # Also create symlinks for original binaries we moved in case they are needed.
     scripts_folder.mkpath
+    libexec.install_symlink scripts_folder => "bin" unless (libexec/"bin").exist?
+    (libexec/"pypybin").each_child do |f|
+      scripts_folder.install_symlink f unless (scripts_folder/f.basename).exist?
```

Comment:
I might need to check what happens on revision bump / upgrade as may be symlinked via Cellar path.